emails/templates/ruled.js

```diff
@@ -9,7 +9,7 @@ const { accountData } = require('../helpers')
 
 module.exports = function() {
   return {
-    subject: 'Dispute {{disputeId}} has been ruled as “{{disputeResult}}”',
+    subject: 'The final ruling in Dispute #{{disputeId}} is “{{disputeResult}}”',
```

Comment:
```suggestion
    subject: 'The final ruling for Dispute #{{disputeId}} is “{{disputeResult}}”',
```

This might sound a bit more natural